deps/nghttp3/lib/nghttp3_ringbuf.c

```diff
@@ -33,21 +33,25 @@
 
 #include "nghttp3_macro.h"
 
-#if defined(_MSC_VER) && defined(_M_ARM64)
-unsigned int __popcnt(unsigned int x) {
+#if defined(_WIN32)
+#  if defined(_M_ARM64)
+unsigned int __nghttp3_popcnt(unsigned int x) {
```

Comment:
I may have missed something but why is this needed? (besides the change of `_MSC_VER` -> `_WIN32`)

Reply:
This is a floating upstream fix. nghttp3 and ngtcp2 both define their own `__popcnt` when building on windows arm64 since that intrinsic is not available. When those are both linked into into the same binary, they would conflict. This commit should be considered temporary as the ngtcp2 author may have another approach in mind to fix it.

Reply:
Oh, for some reason I thought they were file-local but I guess nothing is file-local in C =) thanks for the explanation

Reply:
Yeah, in my original fix I had forgotten that these are C libraries and had made the same assumption :-) ... it didn't help that the build did not fail in my local environment but only in CI!